README.md

```diff
@@ -33,7 +33,8 @@ Here is the Amphion v0.1 demo, whose voice, audio effects, and singing voice are
 )
 
 ## 🚀 News
-- **2024/6/17**: Amphion has a new release for its VALL-E models, it uses Llama as its underlying architecture and has better model performance, faster training speed, and more readable codes compared to our first version. [![readme](https://img.shields.io/badge/README-Key%20Features-blue)](egs/tts/VALLE_V2/README.md)
+- **2024/07/01**: **Emilia** and the **Emilia-Pipe** is now available!
```

Comment:
1. We should use more easy-to-understand words for news. People will be confused because they know little about "Emilia" -- What is this and what is it for? Why is it unique?
2. Use some badges like the following news to link some userful info.